localstripe/localstripe-v3.js

```diff
@@ -80,20 +80,49 @@ function openModal(text, confirmText, cancelText) {
 }
 
 class Element {
-  constructor() {
+  constructor(elements) {
     this.listeners = {};
+    // Element needs a reference to the object that created it, in order to
+    // thoroughly destroy() itself.
+    this._elements = elements;
   }
 
+  // _state tracks calls to unmount, mount, and destroy, and is always one
+  // of 'unmounted', 'mounted', or 'destroyed'.
+  _state = 'unmounted';
```

Comment:
1. This would probably be better inside the constructor, along with other initializations?
2. It's a bit more complex to maintain 3 states represented as string, and I don't feel `destroyed` + warnings "This Element has already been destroyed..." are very useful.
   A simpler solution would be 2 states with something like `this._mounted` (boolean):
   ```js
     constructor(elements) {
       this.listeners = {};
       // Element needs a reference to the object that created it, in order to
       // thoroughly destroy() itself.
       this._elements = elements;
       this._mounted = false;
     }
   ```
   But in my opinion the best would just be an array `this._htmlChildren = []` (described in my review comment).